toronto/src/pages/UserProfile.jsx

```diff
@@ -61,19 +74,25 @@ const UserProfile = () => {
             <ProfileWrapper>
               <Text size={18}>{user.email}</Text>
               <Text size={18}>{user.username}</Text>
-              <StyledLink to={'edit'}>
-                <Button>프로필 편집</Button>
-              </StyledLink>
+              {loginUser ? (
+                userId === loginUser._id ? (
+                  <StyledLink to={'edit'}>
+                    <Button>프로필 편집</Button>
+                  </StyledLink>
+                ) : (
+                  <div></div>
+                )
+              ) : (
+                <div></div>
+              )}
```

Comment:
이 부분도 `loginUser` 가 존재하고 `userId === loginUser._id` 일 때 프로필 편집 버튼을 렌더링 하기 때문에 단축 평가를 사용할 수 있을 것 같습니다!
```suggestion
              {loginUser && userId === loginUser._id && (
                  <StyledLink to={'edit'}>
                    <Button>프로필 편집</Button>
                  </StyledLink>
              )}
```